pkg/retester/retester.go

```diff
@@ -379,6 +395,7 @@ func (c *RetestController) retestOrBackoff(pr tide.PullRequest) error {
 	case retestBackoffPause:
 		c.logger.Infof("%s: %s (%s)", prUrl(pr), "no comment", message)
 	case retestBackoffRetest:
+		retestTotalCounter.With(prometheus.Labels{"org": string(pr.Author.Login), "repo": string(pr.Repository.NameWithOwner)}).Inc()
```

Comment:
This is still too early. Please do it after successfully calling the GH api to create the PR.